tests/plugins/test_store_metadata.py

```diff
@@ -288,6 +289,40 @@ def test_metadata_plugin(tmpdir, br_annotations, expected_br_annotations,
         assert json.loads(annotations['media-types']) == expected_pulp_pull_results
 
 
+@pytest.mark.parametrize(('res'), (
+    {
+        'filesystem-koji-task-id': 'example-fs-taskid',
+        'base-image-id': 'foobar'
```

Comment:
Nitpick: I usually put a comma after each entry in a sequence or dict when they're on separate lines like this, even the last, to make diffs easier to read if we add/remove/swap lines afterwards.